assets/blocks/course-categories-block/course-categories.scss

```diff
@@ -7,8 +7,8 @@
 		padding: 3px 6px;
 		text-decoration: none;
 		border-radius: 2px;
-		margin: 0;
 		white-space: nowrap;
+		margin-bottom: 5px;
```

Comment:
Though it is intended only for the multi-line scenario I think, it is getting applied for all cases. Is it possible have the space only in case of multiline?

Reply:
Considering it depends of the amount of columns more than the screen size, the only way of do it is using 
Container Queries(
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries)
But unfortunately it is only available by < 1% of browsers
https://caniuse.com/css-container-queries.